flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -83,97 +83,97 @@
 	//  Resource Options
 	// ------------------------------------------------------------------------
 
-	public static final ConfigOption<Integer> SQL_RESOURCE_DEFAULT_PARALLELISM =
-			key("sql.resource.default.parallelism")
+	public static final ConfigOption<Integer> TABLE_EXEC_RESOURCE_DEFAULT_PARALLELISM =
+			key("table.exec.resource.default-parallelism")
 					.defaultValue(-1)
 					.withDescription("Default parallelism of job operators. If it is <= 0, use parallelism of StreamExecutionEnvironment(" +
 							"its default value is the num of cpu cores in the client host).");
 
-	public static final ConfigOption<Integer> SQL_RESOURCE_SOURCE_PARALLELISM =
-			key("sql.resource.source.parallelism")
+	public static final ConfigOption<Integer> TABLE_EXEC_RESOURCE_SOURCE_PARALLELISM =
+			key("table.exec.resource.source.parallelism")
```

Comment:
`source-parallelism`, `sink-parallelism`, `external-buffer-memory`, `hash-agg-memory` etc.? A hierarchy with one child is not necessary.